jvm/src/test/kotlin/app/cash/trifle/TrifleTest.kt

```diff
@@ -194,7 +208,8 @@ internal class TrifleTest {
   }
 
   companion object {
+    private val certTTL: Int = 2
```

Comment:
dont need this

Reply:
this constant is being used in multiple places.

Reply:
its only used for 1 unit test though